jdl/parsing/jdl-ast-builder-visitor.js

```diff
@@ -644,5 +652,5 @@ function getSpecialUnaryOptionDeclaration(context, visitor) {
 }
 
 function trimComment(comment) {
-  return comment.replace(/^\/[*]+/, '').replace(/[*]+\/$/, '');
+  return comment.replace(/^\/[*]+[ ]*/, '').replace(/[ ]*[*]+\/$/, '');
```

Comment:
if you want spaces, you can also use \s for readability
or a trim function to replace spaces acting used as prefixes/suffixes
WDYT?

Reply:
\s was replacing multiple whitespace characters like newline, tab etc. Maybe the author would like to have them due to some reason?
But I feel trimming the space is required as the javadoc syntax usually has a space before and after the boundaries `/**` and `*/`.